### ISSUE TITLE
Orient and rotate mitochondrial genome

### ISSUE THREAD

**Issue by sjackman** [tseemann/berokka#25]
I have an animal mitochondrial genome assembled by Unicycler (so no overlap). I'd like to orient and rotate the genome to agree with its closest relative in Genbank. I have a FASTA file of the related genome. Is this task possible with Berokka?

**Comment by tseemann** [comment]
No, I currently have no orientation support.

I'm not even sure if `circlator` can do it for an abritrary DNA sequence - i think it needs an ORF and it does tblastn.

**Comment by sjackman** [comment]
Yeah, I look into circlator too, and came to the same conclusion. I checked and my genome does not start with a coding gene, a tRNA I think (I could confirm).
